test/rocarray/base.jl

```diff
@@ -27,6 +27,31 @@ end
     @test collect(c) == 3:4
 end
 
+@testset "resize!" begin
+    a_h = Array(range(1, 10))
+    a_d = a_h |> roc
+    # Resize up
+    resize!(a_h, 15)
+    resize!(a_d, 15)
+    a_h[10:15] .= 15
+    a_d[10:15] .= 15
+    @allowscalar begin
+        @test a_h == a_d
+    end
+    # Keep the size as is
+    resize!(a_h, 15)
+    resize!(a_d, 15)
+    @allowscalar begin
+        @test a_h == a_d
+    end
+    # Resize down
+    resize!(a_h, 3)
+    resize!(a_d, 3)
+    @allowscalar begin
+        @test a_h == a_d
```

Comment:
As above, test `a_h[1:10] .== Array(a_d[1:10])`

Reply:
Since both arrays will have the length 3, there is no need for slicing.